src/torrent/net/address_info.h

```diff
@@ -52,11 +52,11 @@ ai_clear(addrinfo* ai) {
   std::memset(ai, 0, sizeof(addrinfo));  
 }
 
-inline ai_unique_ptr
+inline std::unique_ptr<addrinfo>
 ai_make_hint(int flags, int family, int socktype) {
-  ai_unique_ptr aip(new addrinfo);
+  std::unique_ptr<addrinfo> aip(new addrinfo);
 
-  aip_clear(aip);
+  ai_clear(aip.get());
```

Comment:
is ai_clear even needed?

Reply:
While correct style, this is a bug fix so no need to change it.